src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java

```diff
@@ -36,6 +39,10 @@
     private String lastName;
 
     @MappingField(columnName = "Official Email")
+    @Email(message = CaseWorkerConstants.INVALID_EMAIL)
+    @Pattern(regexp = CaseWorkerConstants.USER_NAME_PATTERN + "@"
+            + CaseWorkerConstants.DOMAIN_JUSTICE_GOV_UK,
+            message = CaseWorkerConstants.INVALID_EMAIL)
```

Comment:
Added the check on domain. Please let me know if this change needs to be deferred.

Reply:
I think it is failing from UP even though we will use it may create blocker. test if this domain mail id works in aat if not we have to ignore for now.

Reply:
As discussed, justice.gov.uk domain should be used going forward.